porting/deploying-rootfs.md

```diff
@@ -0,0 +1,17 @@
+# Deploying the rootfs and system.img
+
+Once you have built the system.img from android tree, you can download and install the rootfs using halium-install script in [halium-scripts repository](https://github.com/Halium/halium-scripts/).
```

Comment:
from **your** android tree *OR* from **the** android tree

install the rootfs using **the** halium-install script in **the** [halium-scripts repository](https://github.com/Halium/halium-scripts/).